theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/Dot.tsx

```diff
@@ -44,12 +44,12 @@ const Square = styled.div<{isSelected: boolean}>`
 
   z-index: 1;
   pointer-events: none;
-  ${(props) => dims(props.isSelected ? dotSize : dotSize)}
+  ${(props) => dims(props.isSelected ? DOT_SIZE : DOT_SIZE)}
```

Comment:
Change: Give more descriptive name to `dims` and if not already, perhaps document it.

Question: If `dims` is a pure function, then can we just `${dims(DOT_SIZE)}`?

Reply:
I will make this change because yes this makes sense